authentication/views/questions.py

```diff
@@ -26,7 +26,6 @@ class ProfileAnswerViewSet(viewsets.ModelViewSet):
     serializer_class = ProfileAnswerSerializer
     permission_classes = (ProfileAnswerPermission,)
 
-    filter_backends = (SearchFilter, DjangoFilterBackend)
```

Comment:
same

Reply:
Hum I saw `DEFAULT_FILTER_BACKENDS ` has `django_filters.rest_framework.DjangoFilterBackend` so maybe it's ok removing it

Reply:
Indeed.

For the records, if `filter_backends` is declared in a `ViewSet`, the `HasHiddenTagFilter` will *not* work (unless explicitly written of course). So this line removal was actually to make it work.

As `DjangoFilterBackend` is used in almost every view, I think this is a good trade-of.
For `SearchFilter`, I think we will also end up using it everywhere anyway, so if there is no performance downside, we should put it as a default filter too.